docker-x64/Dockerfile

```diff
@@ -52,7 +52,7 @@ RUN git clone --depth=1 https://github.com/pyenv/pyenv.git ${PYENV_ROOT} && \
      pyenv global ${PYTHON_VERSION}
 
 RUN pip install "git+https://github.com/DataDog/datadog-agent-dev.git@${DDA_VERSION}" \
- && dda -v self dep sync -f legacy-build -f legacy-notifications -f legacy-static-quality-gates \
```

Comment:
https://github.com/DataDog/datadog-agent-dev/pull/123/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711

Reply:
Are you saying that the above PR dropped 'legacy-static-quality-gates', so that we must remove them here?
pyproject.toml#L192

Reply:
Yes, the dependency it provided is now required ubiquitously so the feature no longer serves a purpose.